.travis.yml

```diff
@@ -102,7 +103,10 @@ install:
 
   - git clone https://github.com/symengine/symengine symengine-cpp
   - cd symengine-cpp
-  - git checkout 8537c67507fae9b1d8d575d322c6ade4f1a59297
+  #  - git checkout 8537c67507fae9b1d8d575d322c6ade4f1a59297
+  # This PR relies proposed modifications of symengine:
+  # see https://github.com/symengine/symengine/pull/623
+  - git remote add bjodah git://github.com/bjodah/symengine.git; git fetch bjodah; git checkout d9c4e3185a798c4aaeaac75363597e40b10517c9
```

Comment:
Cool, nice way to test things that depend on symengine's PRs that are not yet in.